### DISCUSSION TITLE
0.7.6

### DISCUSSION THREAD

**Discussion by simprecicchiani** [simprecicchiani/ThinkPad-T460s-macOS-OpenCore#138] [General]
- OpenCore and Kexts updated to [December 2021 Release](https://dortania.github.io/hackintosh/updates/2021/12/07/acidanthera-december.html)
- `SecureBootModel` now set to `Disabled`

<hr /><em>This discussion was created from the release <a href='https://github.com/simprecicchiani/ThinkPad-T460s-macOS-OpenCore/releases/tag/0.7.6'>0.7.6</a>.</em>

**Comment by askwakhid** [comment]
Hi,

Thank you for your new EFI.
I have issue with brightness control in Monterey.
This issue like in this video https://youtu.be/OqPL8uT2gMA
I only can adjust brightness level until 50% to 100%, if the brightness level is lower than 50% make screen got black like no screen backlight.

But, If the laptop wake up from sleep mode, the brightness level working normally like a magic. I can adjust brightness level from 100% to 0% normally.

Any suggestion for fix the issue ?
I facing that issue only on Monterey, on Big Sur everything is OK.

Im sorry for my bad English.
Thank you.
